EpikCord/client/websocket_client.py

```diff
@@ -381,17 +381,47 @@ async def _voice_state_update(self, data: discord_typings.VoiceStateUpdateData):
         )  # TODO: Make this return something like (VoiceState, Member) or make VoiceState get Member from member_id
 
     async def _guild_delete(self, data: discord_typings.GuildDeleteData):
-        if guild := self.guilds.remove_from_cache(data["id"]):
+        if guild := self.guilds.remove_from_cache(int(data["id"])):
             await self.dispatch("guild_delete", guild)
 
     async def _interaction_create(self, data: discord_typings.InteractionCreateData):
         interaction = self.utils.interaction_from_type(data)
         await self.dispatch("interaction_create", interaction)
 
     async def _channel_create(self, data: discord_typings.ChannelCreateData):
+        """Event Fired when a channel is created
+
+        :return channel: The Channel which was created
```

Comment:
This docstring is a bit cnfusing to me.

The keyword `return` is used while the function returns nothing, so what is it referring to? It is something with the dispatch argument? Whats about the `:type`? Is it refering to the type of the object passed within the dispatch call, or something else?

i think there is something to change to make this docstring clearer because at the current state it is very confusing. I believe *"Dispatch a `channel_create` event"* or a sphinx marco would do the work better here.

Reply:
You are correct. The return is ~~kind of~~ misleading. Will change the Docstring a

Reply:
Resolved in [b90babe](https://github.com/EpikCord/EpikCord.py/pull/208/commits/b90babe4024d124661ac2321d8cbf4ac2c910978)